models/social_connection.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
:+1:  for being able to use this for other entities. It does seem like everything on FIRST is getting their own social media which we should be able to point to.